### PR TITLE
Project generator

### PR DESCRIPTION
**Pull request by stephanvd**

### CODE REVIEW

File: lib/sequent/generator.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      make_directory
+      copy_files
+      rename_app_file
+      replace_app_name
+    end
+
+    private
+
+    def make_directory
+      FileUtils.mkdir_p(name)
```
Comment (lvonk) [inline lib/sequent/generator.rb]: should we scope this on current_dir. So e.g. `/` etc is allowed?

File: lib/sequent/generator/template_project/db/migrate/1_add_event_store.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
+class AddEventStore < ActiveRecord::Migration[4.2]
```
Comment (lvonk) [inline lib/sequent/generator/template_project/db/migrate/1_add_event_store.rb]: Why is this `4.2`? Also this PR don't test against `4.2` anymore, see travis.yml.

File: lib/sequent/generator/template_project/lib/account/commands.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Account
```
Comment (lvonk) [inline lib/sequent/generator/template_project/lib/account/commands.rb]: Shouldn't this be a `module`?

File: sequent.gemspec [hunk 4]
```diff
@@ -9,6 +9,7 @@ Gem::Specification.new do |s|
   s.authors     = ["Lars Vonk", "Bob Forma", "Erik Rozendaal"]
```
Comment (lvonk) [inline sequent.gemspec]: `svandiepen` :-)

File: spec/lib/sequent/generator_spec.rb [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+require 'fileutils'
+
+describe Sequent::Generator do
+  let(:tmp_path) { 'tmp/sequent-generator-spec' }
+
+  around do |example|
+    FileUtils.rmtree(tmp_path)
+    FileUtils.mkdir_p(tmp_path)
+    Dir.chdir(tmp_path) { example.run }
+    # FileUtils.rmtree(tmp_path)
```
Comment (lvonk) [inline spec/lib/sequent/generator_spec.rb]: Comment

File: lib/sequent/generator/template_project/lib/account/commands.rb (hunk 3 above)
Comment (stephanvd) [reply lib/sequent/generator/template_project/lib/account/commands.rb]: We reopen the class from lib/account.rb here.

**Pull request merged**